roles/kubernetes/node/templates/manifests/kube-proxy.manifest.j2

```diff
@@ -50,7 +50,11 @@ spec:
   volumes:
   - name: ssl-certs-host
     hostPath:
+{% if ansible_os_family == 'RedHat' %}
```

Comment:
Based on [this](https://github.com/kubernetes-incubator/kargo/blob/410438a0e3567d8da38c05899d9383aa07a8d96e/roles/kubernetes/node/tasks/install.yml#L7) should be either `/etc/pki/tls` or `/etc/pki/ca-trust` for RedHat